cardano-cli/cardano-cli.cabal

```diff
@@ -136,21 +141,22 @@ library
                       , ouroboros-consensus-cardano
                       , ouroboros-consensus-protocol
                       , ouroboros-consensus-shelley
-                      , ouroboros-network
+                      , ouroboros-network-api
+                      , ouroboros-network-protocols
```

Comment:
I think we should add bounds here, alternatively you could add a bound on `cardano-api`.

Reply:
This is a problem that inflicts lots of packages.  I'm intending to add them in a separate PR and fix it up for all packages.